infra/base-images/base-builder/detect_repo_test.py

```diff
@@ -75,12 +100,17 @@ def check_commit_with_repo(self, repo_origin, repo_name, commit, tmp_dir):
         'python3', 'detect_repo.py', '--src_dir', tmp_dir, '--example_commit',
         commit
     ]
+    if ref:
+      command.extend(['--ref', ref])
     out, _ = detect_repo.execute(
         command, location=os.path.dirname(os.path.realpath(__file__)))
     match = re.search(r'\bDetected repo: ([^ ]+) ([^ ]+)', out.rstrip())
     if match and match.group(1) and match.group(2):
       self.assertEqual(match.group(1), repo_origin)
       self.assertEqual(match.group(2), repo_name)
+    else:
```

Comment:
what is this part for? why would anyone call this function with `None` repo origin and repo name?